python/ray/util/sgd/data/examples/.gitignore

```diff
@@ -0,0 +1 @@
+ray-project/*
```

Comment:
Is this generated by something? I don't see any reference to this folder at all.

Reply:
This is generated by anyscale. I'm treating this like an IDE artifact

Reply:
Got it